packages/@lwc/template-compiler/src/__tests__/fixtures/renderer-hooks/config/tagname-with-attributes/expected.js

```diff
@@ -0,0 +1,23 @@
+import { parseFragment, registerTemplate, renderer } from "lwc";
+const $fragment1 = parseFragment`<div${3}>Should not be transformed</div>`;
+const stc0 = {
+  city: true,
+};
+function tmpl($api, $cmp, $slotset, $ctx) {
+  const { t: api_text, h: api_element, st: api_static_fragment } = $api;
+  return [
+    api_element(
+      "div",
+      {
+        classMap: stc0,
+        key: 0,
+        renderer: renderer,
+      },
+      [api_text("Should be transformed")]
+    ),
+    api_static_fragment($fragment1(), 2),
```

Comment:
This seems to be untransformed right now due to the static vnode optimization. Maybe we should add a dynamic `{expression}` here to force it to use the non-static vnode.

Reply:
It is actually the other way around: If an element requires a custom renderer, it is not considered static. So in this case both the div can utilize static optimization. But the first div has a renderer hook because it qualifies and the second one does not need a renderer hook. So the static fragment is only applied to the second one. May be i should update the test to say "Should receive a renderer hook" instead of "Should be transformed". Transformed is a generic term.

Reply:
D'oh, you are right. Makes sense.